setup.py

```diff
@@ -19,8 +19,10 @@
     test_suite='tests',
     install_requires=['setuptools',
                       'hyphenation',
-                      'https://github.com/Project-SILPA/pypdflib',
-                      'pyquery',
-                      'Pillow'],
```

Comment:
Gets installed as part of pypdflib. If pypdflib, is added, it will have to be added back